fontforge/sfd.c

```diff
@@ -5834,6 +5841,10 @@ exit(1);
 	    sc->color = temp;
 	} else if ( strmatch(tok,"Comment:")==0 ) {
 	    sc->comment = SFDReadUTF7Str(sfd);
+	} else if ( strmatch(tok,"Decomposition:")==0 ) {
+	    char* decomp = SFDReadUTF7Str(sfd);
+	    sc->user_decomp = utf82u_copy(decomp);
```

Comment:
Limit to 5 chars too?

The other option is to just make it safe to have an arbitrary length